### PR TITLE
CHANGE docs to 3.0.0 && Lint js && UPDATE to use 3.0.0-alpha && FIX webpack

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybooks/storybook/issues/1046

## What I did
- updated all the package references to the new package names
- removed the use of links and actions from `@storybook/react`.
- UPDATED to the latest alpha of storybook
- FIX the webpack config

### CODE REVIEW

**Comment by ndelangen** [comment]
@shilman @UsulPro please review?

please pay attention to the package `gh-pages` being updated, I'm not sure if this will impact anything.

File: stories/data.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ to write great stories.
 * Do not write app-specific code inside your UI components.
 
 ~~~js
-import { linkTo } from @kadira/Storybook
+import { linkTo } from @storybook/react
```
Comment (shilman) [inline stories/data.js]: `import { linkTo } from '@storybook/addon-links';`

**Review by shilman — CHANGES_REQUESTED** [review]
@ndelangen 

I'm getting an error when I try to run:
```
unknown color: rgb(NaN,NaN,NaN)
```

Also it looks like a lot of the CSS has been removed? Does it look OK on your machine when you `yarn develop`?

**Comment by shilman** [comment]
@ndelangen thanks for improving the docs!

two comments:

1) i'm still getting the `unknown color: rgb(NaN,NaN,NaN)` error when i run `yarn develop` -- gatsby is broken

2) let's do the storybook improvements (e.g. adding knobs) in a separate PR, so we can discuss it independently!

**Comment by ndelangen** [comment]
OK, I will continue working on it.

**Comment by ndelangen** [comment]
Should all be fixed now, please check?

File: components/Homepage/Platforms/index.js [hunk 2]
```diff
@@ -6,11 +6,19 @@ const Platform = () => (
     <div className="col-md-12">
       <h3 className="built-for">Built for</h3>
       <p className="platforms">
-        <a href="https://github.com/storybooks/storybook/tree/master/packages/react-storybook" target="_blank">React</a>
+        <a
+          href="https://github.com/storybooks/storybook/tree/master/packages/react-storybook"
```
Comment (shilman) [inline components/Homepage/Platforms/index.js]: https://github.com/storybooks/storybook/tree/master/app/react

File: components/Homepage/Platforms/index.js [hunk 3]
```diff
@@ -6,11 +6,19 @@ const Platform = () => (
...
+          target="_blank"
+        >
+          React
+        </a>
         {' '}
         &
         {' '}
-        <a href="https://github.com/storybooks/storybook/tree/master/packages/react-native-storybook" target="_blank">
+        <a
+          href="https://github.com/storybooks/storybook/tree/master/packages/react-native-storybook"
```
Comment (shilman) [inline components/Homepage/Platforms/index.js]: https://github.com/storybooks/storybook/tree/master/app/react-native

**Review by shilman — CHANGES_REQUESTED** [review]
Minor changes needed, although these are not really related to your change

**Comment by shilman** [comment]
@ndelangen fixed these separately here: https://github.com/storybooks/storybooks.github.io/pull/58

**Review by shilman — APPROVED** [review]

**Pull request closed without merging**